docs/guides/error-handling.mdx

```diff
@@ -1,32 +1,38 @@
 ---
-title: "Retries, Suspensions, and Timeouts"
-description: "Learn how retries, suspensions, and timeouts work in Restate, and how to configure them."
+title: "Error Handling"
+description: "Learn how to handle transient and terminal errors in your applications."
 pagination_next: null
 pagination_prev: null
 ---
 
 import Admonition from "@theme/Admonition";
+import {CodeWithTabs} from "../../src/components/code/code";
 
 # Error Handling
 
+Restate handles retries for failed invocations. By default, Restate does infinite retries with an exponential backoff strategy.
```

Comment:
```suggestion
Restate handles retries for failed invocations. By default, Restate infinitely retries all errors with an exponential backoff strategy.
```

Seems important to underline "all errors" here, given that's how you start the next section